掉落物品识别有时候会出错（截图出现重影，有时会导致崩溃）

### 在提问之前...

- [X] 我理解 Issue 是用于反馈和解决问题的，而非吐槽评论区，将尽可能提供更多信息帮助问题解决
- [X] 我填写了简短且清晰明确的标题，以便开发者在翻阅 issue 列表时能快速确定大致问题。而不是“一个建议”、“卡住了”等
- [X] 我已查看最新测试版本的更新内容，并未提及该 bug 已被修复的情况
- [X] 我已检查了置顶议题（pinned issue），确认我的问题未被提及

### 问题描述

掉落物品识别有时候会出错，发生概率比较低 (<10%).

在出错的情况下，有时候会在 debug 文件夹中出现这种图片（情况一）；极偶尔会出现如下 panic（情况二）

```
terminate called after throwing an instance of 'cv::Exception'
  what():  OpenCV(4.7.0) /home/runner/work/MaaDeps/MaaDeps/vcpkg/buildtrees/opencv4/src/4.7.0-34a2240496.clean/modules/core/src/matrix.cpp:808: error: (-215:Assertion failed) 0 <= roi.x && 0 <= roi.width && roi.x + roi.width <= m.cols && 0 <= roi.y && 0 <= roi.height && roi.y + roi.height <= m.rows in function 'Mat'
```

看起来像是滑动并拼接掉落列表时没有对齐？可能有关的 PR https://github.com/MaaAssistantArknights/MaaAssistantArknights/pull/7202


另外，我这里提供的日志是 panic 时截取的（情况二），此时没有 debug 截图产生。下面的截图是情况一下产生的。

### Version

maa-cli v0.4.5
MaaCore v5.2.1

### 日志文件

[asst.log](https://github.com/MaaAssistantArknights/MaaAssistantArknights/files/15158397/asst.log)


### 截图

![2024-04-28_15-28-58-019_raw](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/12196273/401ea4f8-d36c-4d15-95d8-67ee115d0864)
![2024-04-28_08-27-17-849_raw](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/12196273/807e9669-3d5d-4e0e-bfb3-df8f0fbb904c)
![2024-04-28_00-30-43-624_raw](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/12196273/d02ecfea-ce78-45a8-983d-ade46cc8c380)
g…]()


### 还有别的吗？

_No response_

--- a/src/MaaCore/Task/Fight/StageDropsTaskPlugin.cpp
+++ b/src/MaaCore/Task/Fight/StageDropsTaskPlugin.cpp
@@ -143,7 +143,7 @@ bool asst::StageDropsTaskPlugin::recognize_drops()
 
         // more materials to reveal?
 
-        auto swipe_begin = Point { WindowWidthDefault - 40, 632 };
+        auto swipe_begin = Point { WindowWidthDefault - 240, 632 };
 
         const int swipe_dist = 200;
         ctrler()->swipe(swipe_begin, swipe_begin + swipe_dist * Point::left(), 500, true, 2, 0);

